ksqldb-api-client/src/test/java/io/confluent/ksql/api/client/integration/ClientIntegrationTest.java

```diff
@@ -972,6 +1028,66 @@ public void shouldGetServerInfo() throws Exception {
     assertThat(serverInfo.getKafkaClusterId(), is(expectedClusterId));
   }
 
+  @Test
+  public void shouldListConnectors() throws Exception {
+    // Given:
+    makeKsqlRequest("CREATE SOURCE CONNECTOR " + TEST_CONNECTOR + " WITH ('connector.class'='" + MOCK_SOURCE_CLASS + "');");
+
+    // When:
+    final List<ConnectorInfo> connectors = client.listConnectors().get();
+
+    // Then:
+    assertThat(connectors.size(), is(1));
+    assertThat(connectors.get(0).name(), is(TEST_CONNECTOR));
+    assertThat(connectors.get(0).className(), is(MOCK_SOURCE_CLASS));
+    assertThat(connectors.get(0).state(), is("RUNNING (1/1 tasks RUNNING)"));
+    assertThat(connectors.get(0).type(), is(SOURCE_TYPE));
+  }
+
+  @Test
+  public void shouldDescribeConnector() throws Exception {
+    // Given:
+    makeKsqlRequest("CREATE SOURCE CONNECTOR " + TEST_CONNECTOR + " WITH ('connector.class'='" + MOCK_SOURCE_CLASS + "');");
+
+    // When:
+    final ConnectorDescription connector = client.describeConnector(TEST_CONNECTOR).get();
+
+    // Then:
+    assertThat(connector.type(), is(SOURCE_TYPE));
+    assertThat(connector.state(), is("RUNNING"));
+    assertThat(connector.topics().size(), is(0));
+    assertThat(connector.sources().size(), is(0));
+    assertThat(connector.connectorClass(), is(MOCK_SOURCE_CLASS));
+  }
+
+  @Test
+  public void shouldDropConnector() throws Exception {
+    // Given:
+    makeKsqlRequest("CREATE SOURCE CONNECTOR " + TEST_CONNECTOR + " WITH ('connector.class'='" + MOCK_SOURCE_CLASS + "');");
+
```

Comment:
Add validation that the connector was successfully created into the "Given" here, else the test could be a no-op (as far as the reader is aware).